docs/modules/ROOT/pages/how-to-guides/proc_upgrade_build_pipeline.adoc

```diff
@@ -38,6 +38,17 @@ Confirm that most of the build pipeline tasks that {ProductName} previously skip
 . Select the most recent *PipelineRun*. 
 . View the build pipeline tasks and scroll down to view the vulnerabilities scan, which summarizes the results of the `clair-scan`. 
 
+.Continuous Integration
+
+After upgrading the build pipeline, PipelineRuns will be started to build and test commits in pull requests. The status of the completed PipelineRuns will be visible in the pull requests.
+
+[NOTE]
+====
+{ProductName} sets an expiration date for images produced from pull request commits. The produced images will be deleted from
+quay.io after a week. In order to prevent an image reference from becoming invalid, the image should either be copied to an external
```

Comment:
Default value is 5 days. Also, it's [a parameter](https://github.com/mmorhun/devfile-sample-go-basic/pull/43/files#diff-e2ebdba4f882769a8039c7a84be2da87e275dfbf471a44e12e7d623a6b995ac0R25-R26) and could be adjusted.

Reply:
thanks, I changed to 5 days. If requested, I can document these configurations.

Reply:
I knew I had seen that setting somewhere. 👍